config/accounts.go

```diff
@@ -222,3 +225,50 @@ func (a *AccountIntegration) GetByIntegrationType(integrationType IntegrationTyp
 
 	return integrationEnabled
 }
+
+type AlternateBidderCodes struct {
+	Enabled  bool                                   `mapstructure:"enabled" json:"enabled"`
+	Adapters map[string]AdapterAlternateBidderCodes `mapstructure:"adapters" json:"adapters"`
+}
+
+type AdapterAlternateBidderCodes struct {
+	Enabled            bool     `mapstructure:"enabled" json:"enabled"`
```

Comment:
fyi, w.r.t https://github.com/prebid/prebid-server/pull/2257#discussion_r888823011

> 1. We use `account.alternateBidderCodes.enabled: true/false` as overall toggle for this feature. (default would be false to keep things running as is after upgrade)
> 
>     - If `false`, reject the alternate bidder.
>     - If `true`, check `account.alternateBidderCodes.adapters.BIDDER.allowedBidderCodes`. Here,
>              - If `allowedBidderCodes` is "*", allow the alternate bidder.
>              - Or, if alternate bidder is in `allowedBidderCodes` list, allow the alternate bidder.
>              - Else, if alternate bidder is not in `allowedBidderCodes` list, reject the alternate bidder.
> 2. With this, I don't see the use of `account.alternateBidderCodes.adapters.BIDDER.enabled: true/false`. Since, default empty `allowedBidderCodes` list would act as `account.alternateBidderCodes.adapters.BIDDER.enabled = false`.